network/dag/keys.go

```diff
@@ -44,3 +47,22 @@ func (r NutsKeyResolver) ResolvePublicKey(kid string, sourceTransactionsRefs []h
 
 	return nil, types.ErrNotFound
 }
+
+func (r SourceTXKeyResolver) resolvePublicKey(resolver types.DIDResolver, kid string, metadata types.ResolveMetadata) (crypto.PublicKey, error) {
```

Comment:
not using r, so function instead of method?

Reply:
Could do, yes